### PR TITLE
[BEAM-1362] update release guide post 0.6.0 release

### PR DESCRIPTION
**Pull request by aaltay**
R: @sb2nov @davorbonaci

### CODE REVIEW

**Comment by sb2nov** [comment]
LGTM

**Comment by aaltay** [comment]
retest this please

**Comment by aaltay** [comment]
@davorbonaci Do you have comments on this one?

**Comment by davorbonaci** [comment]
I'll have some comments, but I'll take a look on Monday if possible ;-)

**Comment by aaltay** [comment]
@davorbonaci Can I proceed with this one?

File: src/contribute/release-guide.md [hunk 1]
```diff
@@ -75,7 +75,7 @@ This will list your GPG keys. One of these should reflect your Apache account, f
 
 Here, the key ID is the 8-digit hex string in the `pub` line: `845E6689`.
 
-Now, add your Apache GPG key to the Beam’s `KEYS` file both in [`dev`](https://dist.apache.org/repos/dist/dev/beam/KEYS) and [`release`](https://dist.apache.org/repos/dist/release/beam/KEYS) repositories at `dist.apache.org`. Follow the instructions listed at the top of these files.
+Now, add your Apache GPG key to the Beam’s `KEYS` file both in [`dev`](https://dist.apache.org/repos/dist/dev/beam/KEYS) and [`release`](https://dist.apache.org/repos/dist/release/beam/KEYS) repositories at `dist.apache.org`. Follow the instructions listed at the top of these files. (Note: Not all committers have write access to the [`release`](https://dist.apache.org/repos/dist/release/beam/KEYS) repository. If you end up getting 403 errors ask a PMC member for assistance.)
```
Comment (davorbonaci) [inline src/contribute/release-guide.md]: not all committers -> only PMC members
remove link, no need to repeat it so many times
If you end up getting a 403 error, ask on the mailing list for assistance.

File: src/contribute/release-guide.md [hunk 2]
```diff
@@ -118,6 +118,10 @@ Configure access to the [Apache Nexus repository](http://repository.apache.org/)
 
 Get ready for updating the Beam website by following the [website development instructions]({{ site.baseurl }}/contribute/contribution-guide/#website).
 
+#### Register to PyPI
+
+Release manager needs to have an account with PyPI. If you need one, [register](https://pypi.python.org/pypi?%3Aaction=register_form) at PyPI. You also need to be a maintainer (or an owner) of the [apache-beam](https://pypi.python.org/pypi/apache-beam) package in order to push a new release. Ask a PMC member to add you as a maintainer.
```
Comment (davorbonaci) [inline src/contribute/release-guide.md]: Ask on the mailing list for assistance.

File: src/contribute/release-guide.md [hunk 3]
```diff
@@ -210,6 +214,19 @@ Check out the release branch.
 
 The rest of this guide assumes that commands are run in the root of a repository on `${BRANCH_NAME}` with the above environment variables set.
 
+### Update the Python SDK version
+
+In the master branch update Python SDK [version](https://github.com/apache/beam/blob/master/sdks/python/apache_beam/version.py) to the next development version (e.g. `1.2.3.dev` to `1.3.0.dev`).
```
Comment (davorbonaci) [inline src/contribute/release-guide.md]: In the master branch, update the Python SDK version identifier to the next development version.

File: src/contribute/release-guide.md [hunk 4]
```diff
@@ -210,6 +214,19 @@ Check out the release branch.
...
+
+In the release branch update the Python SDK version to the release version (e.g. `1.2.3.dev` to `1.2.3`).
```
Comment (davorbonaci) [inline src/contribute/release-guide.md]: In the release branch, update

File: src/contribute/release-guide.md [hunk 5]
```diff
@@ -343,6 +377,7 @@ Start the review-and-vote thread on the dev@ mailing list. Here’s an email tem
     * all artifacts to be deployed to the Maven Central Repository [4],
     * source code tag "v1.2.3-RC3" [5],
     * website pull request listing the release and publishing the API reference manual [6].
+    * python artifacts are deployed along with the source release to to dist.apache.org [2].
```
Comment (davorbonaci) [inline src/contribute/release-guide.md]: to the
python -> Python

File: src/contribute/release-guide.md [hunk 6]
```diff
@@ -404,6 +439,13 @@ Once the release candidate has been reviewed and approved by the community, the
 
 Use the Apache Nexus repository to release the staged binary artifacts to the Maven Central repository. In the `Staging Repositories` section, find the relevant release candidate `orgapachebeam-XXX` entry and click `Release`. Drop all other release candidates that are not being released.
 
+### Deploy Python artifacts to PyPI
+
+1. Create a new release and upload python zip file for the new release using [PyPI UI] (https://pypi.python.org/pypi/apache-beam)
```
Comment (davorbonaci) [inline src/contribute/release-guide.md]: the Python zip file
the PyPI UI

File: src/contribute/release-guide.md [hunk 7]
```diff
@@ -404,6 +439,13 @@ Once the release candidate has been reviewed and approved by the community, the
...
+1. Alternatively use the command line tool to upload the new release `twine upload apache-beam-${VERSION}.zip`
```
Comment (davorbonaci) [inline src/contribute/release-guide.md]: Alternatively,

File: src/contribute/release-guide.md [hunk 8]
```diff
@@ -426,9 +468,10 @@ In JIRA, inside [version management](https://issues.apache.org/jira/plugins/serv
 
 * Maven artifacts released and indexed in the [Maven Central Repository](https://search.maven.org/#search%7Cga%7C1%7Cg%3A%22org.apache.beam%22)
 * Source distribution available in the release repository of [dist.apache.org](https://dist.apache.org/repos/dist/release/beam/)
+* Source distribution removed from the dev repository of [dist.apache.org](https://dist.apache.org/repos/dist/dev/beam/)
 * Website pull request to [list the release]({{ site.baseurl }}/use/releases/) and publish the [API reference manual]({{ site.baseurl }}/documentation/sdks/javadoc/) merged
 * Release tagged in the source code repository
-* Release version finalized in JIRA
+* Release version finalized in JIRA. (Note: Not all committers have administrator access to JIRA. If you end up getting permissions errors ask a PMC member for assistance.)
```
Comment (davorbonaci) [inline src/contribute/release-guide.md]: mailing list

**Comment by aaltay** [comment]
Thank you @davorbonaci. Updated PR, PTAL.

**Pull request closed without merging**